### DISCUSSION TITLE
테스트등록 상태에서 이벤트 훅 설정 수정이 안됩니다.

### DISCUSSION THREAD

**Discussion by wileyday** [commerce-api-naver/commerce-solutions#90] [묻고 답하기] [answered]
안녕하세요?

아래와 같이 이벤트 훅 설정 입력을 하고 하단에 저장하기를 눌러 저장을 마쳤습니다.

![image](https://github.com/user-attachments/assets/4eb01f58-9310-40e0-8420-eaf3f28fb70b)

이 과정을 컴퓨터 바꿔가며 여러차례 해보았지만 이벤트 훅 설정이 미사용으로 되돌아갑니다.

![image](https://github.com/user-attachments/assets/805f67b2-b4a6-4569-a907-1007bd51d73c)

테스트등록 상태라서 이벤트 훅 설정이 불가한 것일까요?

**Comment by commerce-api-naver** [comment, marked as answer]
안녕하세요. 커머스API 이민석입니다.

이벤트 훅은 테스트용 검증 후 수신받으실 이벤트 유형(예: 솔루션 추가 - "START_SUBSCRIPTION")까지 지정해야 저장됩니다.
입력하신 정보로 실제 이벤트 훅 처리가 가능할 정도로 연동 개발을 완료하신 뒤 커머스API센터에서 수신 정보 입력 후 우측 하단 [검증] 버튼을 선택하여 입력 내용의 실제 호출 여부 확인과 수신 이벤트 유형까지 지정하신 이후 변경 정보를 저장해보시기 바랍니다.

**Reply by wileyday** [reply]
[검증] 버튼도 눌러 봤었는데 아래쪽으로 아무것도 출력되지 않아서 그냥 저정되지 싶었는데 그게 아니었네요.
문서를 자세히 읽어보니 반드시 HTTP CODE 200 을 리턴해야 한다고 되어 있어서 (기존에는 201로 리턴) 수정하고 [검증] 버튼 눌렀더니,
말씀처럼 아래와 같이 이벤트 유형 목록이 출력됩니다! 감사합니다 ^^

![image](https://github.com/user-attachments/assets/cac34bea-de53-4f89-a760-be78e26632c8)
